site/content/en/docs/Guides/feature-stages.md

```diff
@@ -26,12 +26,20 @@ The current set of `alpha` and `beta` feature gates are:
 
 | Feature Name | Gate    | Default | Stage | Since |
 |--------------|---------|---------|-------|-------|
+{{% feature expiryVersion="1.10.0" %}}
```

Comment:
This would break the table. Let's do a separate Revert + New version. WDYT?

Reply:
Done